MEN-4180: improve onboarding tooltips visibility

tranchitella opened the pull request.

mzedel reviewed (changes requested):
👍 nicely done, the fallback approach is just what I had in mind! if I remember correctly there were a few more steps that served as an anchor though (or we will have to reopen the issue very soon)...

tranchitella commented on src/js/actions/onboardingActions.js:
```diff
@@ -31,7 +31,7 @@ export const getOnboardingState = () => (dispatch, getState) => {
         ? store.devices.byId[acceptedDevices[0]].attributes.device_type
         : null;
-    const progress = onboardingState.progress || determineProgress(acceptedDevices, pendingDevices, releases, pastDeployments);
+    const progress = applyOnboardingFallbacks(onboardingState.progress) || determineProgress(acceptedDevices, pendingDevices, releases, pastDeployments);
```
Ack

tranchitella commented:
@mzedel ready for review; I've incorporated your changes to the fallback, but also added three commits (you can review them, specifically) to streamline the onboarding tips. It works well here, now. No more spurious welcomeTip at application startup, which was annoying me so much.

mzedel reviewed (approved):
Thank you very much for the fixes! 🎉 especially for the welcometip!

mzedel commented:
merging to get this straight to staging...
